GameData/RealismOverhaul/TestFlight_Generic_Engines.cfg

```diff
@@ -1,3 +1,55 @@
+// Pre-process: EngineConfigs/ put TESTFLIGHT nodes inside MEC/CONFIGs;
+// this lets us copy some fields from one to the other, but then the
+// FOR:zTestFlight processing steps need them directly in the PART node.
+// Must use BEFORE to preserve TestLite compatibility.
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]]:BEFORE[zTestFlight]
```

Comment:
All instances of ModuleEngineConfigs should be replaced by Module*EngineConfigs so ModuleBimodalEngineConfigs also gets included in the patching.

Reply:
done; and confirmed LR129 has a working TF config again